docs/docs/legal/policies/acceptable-use-policy.md

```diff
@@ -17,13 +17,28 @@ You may not:
 4. Attempt to probe, scan, penetrate or test the vulnerability of our Subscription Services, Website, systems, or network or try to circumvent our authentication. Any penetration testing must not be conducted without prior written consent by CAOS.
 5. Use any organization or domain name that includes or is confusingly similar with trademarks, or any third parties. CAOS may determine any violation at its sole discretion
 6. Collecting any information about our Customers, our Customers users, or our users without the consent of the person identified. This includes phishing, social engineering, scamming, spidering or harvesting information from any Subscription Service or Website
+7. Use Actions to run workloads that are unrelated to the Subscription Services and Websites, such as excessively calling unrelated third party services, crypto mining, intentionally long running code
 
-## Fair Use Principle
+## Fair use principles
 
-The “fair use” principle applies to the use of our services. We optimize our infrastructure in such a way that sufficient capacity is available to you even during short-term increased demand (“peaks”) and implement mitigation measures such as our [Rate Limit Policy](rate-limit-policy).  You are nonetheless required to adhere to reasonable use of our resources in order to avoid negatively affecting the services for other customers.
+The “fair use” principle applies to the use of our services.
+We optimize our infrastructure in such a way that sufficient capacity is available to you even during short-term increased demand (“peaks”) and implement mitigation measures such as our [Rate Limit Policy](rate-limit-policy).
+You are nonetheless required to adhere to reasonable use of our resources in order to avoid negatively affecting the services for other customers.
 
 You agree that we may delete any data on our systems or networks, if CAOS believes that this data may corrupt our systems, interfere or may compromise other customers' data.
 
+You agree to adhere to the following fair use limits:
+
+- [Actions minutes](../service-description/billing.md#action-minutes): A monthly limit of 1'000 Action minutes per instance
+- Usage limits that were agreed by both parties in advance for the duration of the term
+
 ## Violations of this policy
 
-We may suspend or terminate your usage of our Services for any violation of this Acceptable Use Policy. You will not be entitled to any Financial Credit or compensation for any interruptions caused by violation of this policy.
+We may suspend or terminate your usage of our Services for any violation of this Acceptable Use Policy.
+You will not be entitled to any Financial Credit or compensation for any interruptions caused by violation of this policy.
+
+## Entry into force
+
+This policy is valid from November 9, 2023.
+
+Last revised November 9, 2023
```

Comment:
Did this policy not already exist?

Would not the last revised only need to be changed?